core/controllers/creator_dashboard.py

```diff
@@ -264,20 +248,20 @@ class NotificationsHandler(base.BaseHandler):
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
+    @acl_decorators.can_access_creator_dashboard
```

Comment:
@1995YogeshSharma -- did you check when this handler gets called? Is it called when the user is logged out?